services/gatestext_script.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	"fmt"
+	"time"
+)
+
+func main() {
+	fmt.Printf("GATESTEXT SERVICE STARTED")
+
+	for true {
+		time.Sleep(1 * time.Second)
```

Comment:
1 sec might be too verbose for a long running VM (incase you ever wanna debug or something). Maybe increase it to 2-5 mins?